internal/host/transition.go

```diff
@@ -138,6 +139,28 @@ func (th *transitionHandler) PostRegisterDuringInstallation(sw stateswitch.State
 		"The host unexpectedly restarted during the installation")
 }
 
+func (th *transitionHandler) PostRegisterAfterInstallation(sw stateswitch.StateSwitch, args stateswitch.TransitionArgs) error {
+	_, ok := sw.(*stateHost)
+	if !ok {
+		return errors.New("RegisterNewHost incompatible type of StateSwitch")
+	}
+	_, ok = args.(*TransitionArgsRegisterHost)
+	if !ok {
+		return errors.New("PostRegisterAfterInstallation invalid argument")
+	}
+
+	return common.NewApiError(
+		http.StatusForbidden,
+		errors.New(
+			"Host is trying to register after the cluster has already been installed. "+
```

Comment:
@jhernand if the user will see this message in the UI - @nirfarkas should review it

Reply:
Thanks @gamli75, I forgot that. The message will be part of an error event, so it is displayed to the user. @nirfarkas please review.

Reply:
@jhernand what do you mean "error event"? Did you verify that we don't spam user events?

Reply:
I mean this:

![image](https://user-images.githubusercontent.com/1423971/203735037-fcd7145f-3394-4c60-ac81-738aa3c505d1.png)

That event will be generated only once because the agent will then freeze.